lightwood/helpers/ts.py

```diff
@@ -29,18 +30,20 @@ def get_inferred_timestamps(df: pd.DataFrame, col: str, deltas: dict, tss: Times
 def add_tn_conf_bounds(data: pd.DataFrame, tss_args: TimeseriesSettings):
     """
     Add confidence (and bounds if applicable) to t+n predictions, for n>1
-    @TODO: active research question: how to guarantee 1-e coverage for t+n, n>1
-    for now, we replicate the width and conf obtained for t+1
+    TODO: active research question: how to guarantee 1-e coverage for t+n, n>1
+    For now, conservative approach is to increase width by the confidence times the log of the time step.
     """
     for col in ['confidence', 'lower', 'upper']:
         data[col] = data[col].astype(object)
 
     for idx, row in data.iterrows():
+        error_increase = [row['confidence']] + [row['confidence'] * np.log(np.e + t / 2)
```

Comment:
what is np.e? can you document why this function is as such?

Reply:
The numerical constant `e`. Will improve the reasoning for this in the docstring.